apps/src/common.rs

```diff
@@ -1276,6 +1304,8 @@ impl HttpConn for Http3Conn {
 
                 Ok((_stream_id, quiche::h3::Event::Finished)) => {
                     self.reqs_complete += 1;
+                    let _ = self.reqs_in_flight.saturating_sub(1);
```

Comment:
```suggestion
                    self.reqs_in_flight = self.reqs_in_flight.saturating_sub(1);
```

Need to assign the value or else it won't do anything